renovate.json

```diff
@@ -3,7 +3,7 @@
 	"packageRules": [
 		{
 			"paths": [ "packages/**" ],
-			"packagePatterns": ["*"],
+			"packagePatterns": [ "*" ],
```

Comment:
@blowery Should I worry about changes like this and all above from `client/` folder? I didn't even touch these files. At least I didn't do it on purpose :)